aci/resource_aci_rtctrlattrp.go

```diff
@@ -128,6 +324,96 @@ func resourceAciActionRuleProfileCreate(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(err)
 	}
 
+	// rtctrlSetPref - Operations
+	if setPref, ok := d.GetOk("set_preference"); ok {
+
+		log.Printf("[DEBUG] rtctrlSetPref: Beginning Creation")
+
+		rtctrlSetPrefAttr := models.RtctrlSetPrefAttributes{}
+		rtctrlSetPrefAttr.LocalPref = setPref.(string)
+		rtctrlSetPref := models.NewRtctrlSetPref(fmt.Sprintf(models.RnrtctrlSetPref), rtctrlAttrP.DistinguishedName, "", "", rtctrlSetPrefAttr)
+
+		creation_err := aciClient.Save(rtctrlSetPref)
+		if creation_err != nil {
+			return diag.FromErr(creation_err)
+		}
+
+		log.Printf("[DEBUG] %s: Creation finished successfully", rtctrlSetPref.DistinguishedName)
+		resourceAciRtctrlSetPrefRead(ctx, rtctrlSetPref.DistinguishedName, d, m)
```

Comment:
Why do a read specific for this sub-option when in the main read at the end you also re-read the sub-option? Aren't you doing two read in a row? (Same comment for all other sub-option)